Framework/PythonInterface/plugins/algorithms/WorkflowAlgorithms/MSDFit.py

```diff
@@ -31,7 +31,8 @@ def PyInit(self):
         self.declareProperty(MatrixWorkspaceProperty('InputWorkspace', '', direction=Direction.Input),
                              doc='Sample input workspace')
         self.declareProperty(name='Model', defaultValue='Gauss',
-                             validator=StringListValidator(['Gauss', 'Peters', 'Yi']),
+                             validator=StringListValidator(
+                                 ['Gauss', 'Peters', 'Yi']),
```

Comment:
I think the line length in Python is 130 or 140 characters. In the future, make sure to use `--config=.flake8` when running `flake8` in the root of Mantid's source tree to use our settings.

Otherwise this is OK, no need to undo these formatting changes.